packages/front/src/lib/components/Documents/Documents.controller.ts

```diff
@@ -40,18 +40,44 @@ export class DocumentsController {
     zipPromise: Store<Promise<void | null>>;
     public showMoreAsso: Store<boolean>;
     public showMoreEstab: Store<boolean>;
+    public selectedDocsOrNull: Store<{
+        assoDocs: (DocumentEntity | undefined)[];
+        estabDocs: (DocumentEntity | undefined)[];
+        moreAssoDocs: (DocumentEntity | undefined)[];
+        moreEstabDocs: (DocumentEntity | undefined)[];
+    }>;
+    public flatSelectedDocs: ReadStore<DocumentEntity[]>;
+    private identifier: string;
+    public downloadBtnLabel: ReadStore<string>;
 
     constructor(
         public resourceType: ResourceType,
         // TODO: replace unknown with EstablishmentEntity when created
         public resource: AssociationEntity | unknown,
     ) {
+        // @ts-expect-error -- missing type
```

Comment:
Pourquoi qu'on a besoin de ca ?

Reply:
`resource` peut être `unknown` parce qu'on n'a pas fait de `EstablishmentEntity` côté front encore

Reply:
On peut avoir un Record<string, string> non ? Si on sait que c'est forcément un objet plat.

Mais je suis pas sur de comprendre ta réponse, le ts-exprect-error est bien pour le this.identifier ? Si c'est le cas soit on dit que indentifier peut-etre undefined ou string soit dans le constructeur on throw une erreur si on est pas capable de trouver une string. Je préfère qu'on évite les ts-expect-error quand il suffit de checker si la valeur est bien présente.

Reply:
non non il est pour les resource.rna, resource.siren et resource.siret. Si on remplace `unknown` par `Record<string, string>` ça va pas, parce que les établissements c'est pas plat, et parce que même si ça l'était il trouve pas spécifiquement ces attributs